.circleci/config.yml

```diff
@@ -12,76 +12,183 @@ version: 2.1
 orbs:
   win: circleci/windows@2.2.0
   sfchangecase: salesforce/change-case-management@3
+  slack: circleci/slack@3.4.2
 
-_refs:
-  defaults: &defaults
-    working_directory: ~/cli
-    docker:
-      - image: circleci/node:12.4.0
-  ssh-config: &ssh-config
-    fingerprints:
-      - '3b:68:69:7d:19:a4:50:69:ad:b8:db:57:f9:9c:f0:3f'
-  restore_cache: &restore_cache
-    keys:
-      - dependency-cache-v3-{{ checksum "package.json" }}
-      # if cache for exact version of 'package.json' is not present, then load the most recent cache
-      - dependency-cache-v3-
-  install: &install
-    name: Install dependencies
-    command: |
-      npm --version
-      node --version
-      npm install
-  gh-config: &gh-config
-    name: Configuring GitHub
-    command: |
-      git config credential.helper 'cache --timeout=120'
-      git config user.email "$GH_EMAIL"
-      git config user.name "Release Bot"
-  gus-prepare-environment-variables: &gus-prepare-environment-variables
-    name: 'Prepare environment variables for GUS Change Case Management'
-    command: |
-      export GUS_BUILD="$(node -pe "require('./packages/salesforcedx-vscode/package.json').version")"
-      echo "export GUS_RELEASE=offcore.tooling.$GUS_BUILD" >> $BASH_ENV
+parameters:
```

Comment:
These are circleci pipeline parameters. What makes this special is that we can change how the workflow behaves depending on the parameters that we pass in to the circleci API call. What this means is that we can easily run with a different version of node, the cli, trigger a creation of the release, and publish with more flexibility and control through circle ci.